azurelinuxagent/common/protocol/goal_state.py

```diff
@@ -303,6 +317,7 @@ def _fetch_full_wire_server_goal_state(self, incarnation, xml_doc):
                 # Note that we do not save the certificates to the goal state history
```

Comment:
Now we save the certificate to history folder, should we remove this comment?

Reply:
we do not save the certificate to the history folder, we never should do that since it is a secret. we only save the thrumbprint, which can be helpful when debugging

Reply:
ah, i see the confusion now... i'll update the comment to say we save the summary